src/components/topbar/Topbar.jsx

```diff
@@ -24,27 +28,70 @@ const Topbar = (props) => {
+          </a>
+        </Col>
+      </Row>
+    </div>
+  );
   return (
     <div>
       <div className="topbar">
-        <Icon type="menu" className="hamburger" onClick={()=>store.dispatch(set("uiState.showSidenav", true))}/>
-        <img className="logo" src={logo} alt="logo" />
-        <img className="upLogo" src={upLogo} alt="logo" />
-        {props.showProjectSelector && <div className="btn-position">
-          <Button className="action-rounded" onClick={() => setModalVisible(true)}>{projectName}
-            <Icon type="caret-down" />
-          </Button>
-        </div>}
-        {props.showDbSelector && <div className="db-btn-position">
-          <Button className="action-rounded" onClick={() => setVisible(true)}>
-            <img src={svgIcon} alt={selectedDB} style={{ marginRight: 10 }} />
-            {selectedDB}
-            <Icon type="caret-down" />
-          </Button>
-        </div>}
-        <DbSelector visible={visible} handleSelect={handleDBSelect} handleCancel={() => setVisible(false)} />
+        
+          <Icon type="menu" className="hamburger" onClick={()=>store.dispatch(set("uiState.showSidenav", true))}/>
+          <img className="logo" src={logo} alt="logo" />
+          <img className="upLogo" src={upLogo} alt="logo" />
+          {props.showProjectSelector && <div className="btn-position">
+            <Button className="action-rounded" onClick={() => setModalVisible(true)}>{projectName}
+              <Icon type="caret-down" />
+            </Button>
+          </div>}
+          {props.showDbSelector && <div className="db-btn-position">
+            <Button className="action-rounded" onClick={() => setVisible(true)}>
+              <img src={svgIcon} alt={selectedDB} style={{ marginRight: 10 }} />
+              {selectedDB}
+              <Icon type="caret-down" />
+            </Button>
+          </div>}
+          
+          <DbSelector visible={visible} handleSelect={handleDBSelect} handleCancel={() => setVisible(false)} />
 
-        <SelectProject visible={modalVisible} handleCancel={() => setModalVisible(false)} />
+          <SelectProject visible={modalVisible} handleCancel={() => setModalVisible(false)} />
+        <div className="right-list">
+          <Menu mode="horizontal">
+            <Menu.Item>
+              <a href="https://docs.spaceuptech.com/" target="_blank">Docs</a>
+            </Menu.Item>
+            <Menu.Item>
+              <a href="https://learn.spaceuptech.com/" target="_blank">Learn</a>
+            </Menu.Item>
+            <Menu.Item>
+              <a href="https://github.com/spaceuptech/space-cloud" target="_blank">
+                <img src={githubIcon} />
+              </a>
+            </Menu.Item>
+            <Divider type="vertical" style={{height:"40px"}}/>
+            <Menu.Item>
+              <Popover content={content} trigger="click" placement="bottom">
```

Comment:
Lets keep the placement as `bottomRight`. We want the arrow at the corner pointing the heart shape